Structure_Engine/Modify/SetAnalysisParameters.cs

```diff
@@ -82,7 +82,7 @@ public static void SetAnalysisParameters(this SawnTimber timber, TimberAnalysisS
                 timber.YoungsModulus = new Vector { X = timber.E_0_k, Y = timber.E_90_Mean, Z = timber.E_90_Mean };
                 timber.ShearModulus = new Vector { X = timber.G_Mean, Y = timber.G_Mean, Z = timber.G_Mean };
 
-                message = $"{nameof(SawnTimber)} materials does not contain characteristic values for youngs modulus perpendicular. Mean value of perpendicular stiffness used." + Environment.NewLine;
+                message = $"{nameof(SawnTimber)} materials does not contain characteristic values for youngs modulus perpendicular. Mean value of perpendicular stiffness used." + System.Environment.NewLine;
```

Comment:
This suggests the Environment namespace has somehow started to bleed into Structures if the compiler was complaining about this? If so we should investigate and prevent

Reply:
It does, but I don't understand how?

Reply:
Fixed itself, change removed now.